quickwit/quickwit-query/src/query_ast/term_query.rs

```diff
@@ -20,9 +20,10 @@
 use std::collections::HashMap;
 
 use serde::{Deserialize, Serialize};
-use tantivy::schema::Schema as TantivySchema;
+use tantivy::schema::{FieldType, Schema as TantivySchema};
 
 use super::{BuildTantivyAst, QueryAst};
+use crate::query_ast::utils::DYNAMIC_FIELD_NAME;
```

Comment:
`term_query.rs` should not have been modified at all.

This PR is obfuscating what it does with logic that only concerns term_set_query.